app/patients/delete/template.hbs

```diff
@@ -5,6 +5,6 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    {{t 'patients.messages.deletePatient' firstName=model.firstName lastName=model.lastName}}
+    {{t 'messages.delete' name=(concat model.firstName " " model.lastName)}}
```

Comment:
This should user model.shortDisplayName instead of concatenating first and lastName